### PR TITLE
feat(language): add Italian translation

### PR DESCRIPTION
**Pull request by allanlewis**
This PR adds an Italian language package.

Please let me know if I've missed anything!

### CODE REVIEW

File: packages/languages/it/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "module": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
+  "description": "The Italian language package for zxcvbn-ts",
+  "license": "MIT",
+  "homepage": "https://github.com/zxcvbn-ts/zxcvbn",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zxcvbn-ts/zxcvbn.git"
+  },
+  "author": "zxcvbn-ts",
```
Comment (MrWook) [inline packages/languages/it/package.json:14]: If you like you can be the author of it :)

**Comment by allanlewis** [comment]
Thanks for the comments! I've addressed them and rebased on master.

**Comment by allanlewis** [comment]
Thanks for the quick responses, @MrWook 👏

**Comment by allanlewis** [comment]
@MrWook if you could post here, or otherwise let me know, when the package is published, that would be great 🙏

**Comment by MrWook** [comment]
@allanlewis thanks for your contribution ❤️ , i published it https://www.npmjs.com/package/@zxcvbn-ts/language-it

**Comment by allanlewis** [comment]
> i published it

Thanks!

**Pull request merged**